proteus/mprans/SedClosure.h

```diff
@@ -576,7 +576,7 @@ namespace proteus
 
     
 
-    inline double*  mIntFluid(  double sedF,
+    inline void  mIntFluid( double * mint2, double sedF,
```

Comment:
Correct. . I will merge for now but I will update all of them later (I did the same in WaveTools.h)